spiketoolkit/sortingcomponents/detection.py

```diff
@@ -19,17 +19,16 @@ def detect_spikes(recording, channel_ids=None, detect_threshold=5, n_pad_ms=2, u
     channel_ids: list or None
         List of channels to perform detection. If None all channels are used
     detect_threshold: float
-        Threshold in MAD to detect peaks
-    n_pad_ms: float
-        Time in ms to find absolute peak around detected peak
-    upsample: int
-        The detected waveforms are upsampled 'upsample' times (default=1)
+        Threshold in median absolute deviations (MAD) to detect peaks
+    n_shifts: int
+        Number of shifts to find peak. E.g. if n_shift is 2, a peak is detected (if detect_sign is 'negative') if
+        two samples before the peaks are decreasing and two sampels after the peak are increasing.
```

Comment:
Not exactly true : I would put. a local extrema is more/less than 2_shifts on each side.